packages/create-catalyst/src/commands/create.ts

```diff
@@ -71,8 +71,6 @@ export const create = async (options: CreateCommandOptions) => {
       canCreateChannel = false;
     } else if (channels.length >= limits.total_including_inactive) {
       canCreateChannel = false;
-    } else {
-      console.log(`${limits.active - activeChannels.length} active channels remaining.`);
     }
```

Comment:
As-is, this log is leading to more confusion than anticipated. I will rework this + the logic above as part of CATALYST-313

Reply:
🍹  It would be cleaner if this was its own commit, but given that you're the main contributor at the moment it's probably fine.